### PR TITLE
Programmed actions UI

### PR DESCRIPTION
**Pull request by jenf**
This is built on top of #3796, hopefully it'll update assuming that goes in

### CODE REVIEW

File: lib/engine/game/g_18_mag.rb [hunk 1]
```diff
@@ -3,6 +3,9 @@
 module Engine
   module Game
     module G18Mag
+      def available_programmed_actions
```
Comment (ryandriskel) [inline lib/engine/game/g_18_mag.rb:6]: I don't think this is the right place for this.

Comment (jenf) [reply lib/engine/game/g_18_mag.rb:6]: No, I'll fix after Toby reviews

Comment (roseundy) [reply lib/engine/game/g_18_mag.rb:6]: Kinda confused why 18Mag is in here at all.

**Review by tobymao — APPROVED** [review]

File: lib/engine/game/g_18_mag.rb (hunk 1 above)
Comment (jenf) [reply lib/engine/game/g_18_mag.rb:6]: 18mag does actually float corps

**Pull request merged**